examples/ct/cert_checker_test.go

```diff
@@ -94,8 +94,7 @@ func TestCertCheckerInvalidChainAccepted(t *testing.T) {
 		t.Fatal("failed to load fake root")
 	}
 
-	_, err := ValidateChain(jsonChain, *trustedRoots, false)
-
+	_, err := ValidateChain(jsonChain, CertValidationOpts{trustedRoots, false, []x509.ExtKeyUsage{x509.ExtKeyUsageAny}})
```

Comment:
(nit) Maybe have a `testValidationOpts` at the top of the file, with field names broken out, and use it throughout?